### PR TITLE
Added improved designs for OG images.

### PR DESCRIPTION
**Pull request by JeroenReumkens**
## What does this PR do?

Adds updated designs for Apps and Meeting OG Images.

### Why draft? 
Because it's not 100% according to design yet, and Peer needs to decide whether he wants to give this a go or let me tweak this still a bit.

### Screenshots

image

image

image

image

image

**Environment**: Staging(main branch)

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

## How should this be tested?

- [ ] Go to meeting and app pages and check the images in the `og:image` meta tag and ensure they look nice <3

### CODE REVIEW

File: packages/app-store/riverside/config.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
   "name": "Riverside",
   "slug": "riverside",
   "type": "riverside_video",
-  "imageSrc": "/api/app-store/riverside/icon.svg",
-  "logo": "/api/app-store/riverside/icon.svg",
+  "imageSrc": "/api/app-store/riverside/icon-dark.svg",
```
Comment (JeroenReumkens) [inline packages/app-store/riverside/config.json:6]: We've decided to add "dark" to a few icons which are mainly of a dark color. In the render of the OG images we check this name and don't render a gradient blob in the background if it's a dark image.

File: packages/lib/OgImages.tsx [hunk 2]
```diff
@@ -127,46 +127,77 @@ export const Meeting = ({ title, users = [], profile }: MeetingImageProps) => {
...
         </div>
       </div>
     </Wrapper>
   );
 };
 
+const VisualBlur = ({ visualSlug }: { visualSlug: string }) => {
+  // Making a blur of a dark logo is very ugly. We use the filename to indicate,
+  // when we don't want to render these blurry blob backgrounds.
+  if (visualSlug.indexOf("dark") > -1) return null;
```
Comment (JeroenReumkens) [inline packages/lib/OgImages.tsx:149]: This checks if it's a dark image and doesn't render a gradient blob.

**Review by PeerRich — APPROVED** [review]
this is awesome, great fixes!

only NIT: the "more" circle is not the same size and the "+2" is not horizontally, vertically centered
CleanShot 2022-10-19 at 17 00 00@2x

**Comment by JeroenReumkens** [comment]
@PeerRich  Thanks! 🙏  you want to have the 'more' circle the same size as the avatars you mean?

**Comment by JeroenReumkens** [comment]
@PeerRich  updated the size of the circle. Also offset the +2 a little bit to the top and left — it was actually centered before, so that was an optical illusion. But I think it now visually looks like it's centered better, while it actually isn't anymore, haha. 

image

**Review by zomars — APPROVED** [review]
LGTM

**Pull request merged**